libs/blocks/nav-marquee/nav-marquee.css

```diff
@@ -0,0 +1,278 @@
+.nav-marquee {
+  flex: 1 1 auto;
+}
+
+.nav-marquee.light {
```

Comment:
What is the default color-scheme on this? Dark or light. Which ever one is default shouldn't need a helper `.light` class.

Reply:
Default is light, so I will remove helper